scripts/generate-f32-gemm.sh

```diff
@@ -621,6 +621,11 @@ tools/xngen src/f32-gemm/avx512-broadcast.c.in -D MR=7 -D NR=16 -D INC=1 -D DATA
 tools/xngen src/f32-gemm/avx512-broadcast.c.in -D MR=8 -D NR=16 -D INC=0 -D DATATYPE=F32 -o src/f32-gemm/gen/f32-gemm-8x16-minmax-avx512f-broadcast.c &
 tools/xngen src/f32-gemm/avx512-broadcast.c.in -D MR=8 -D NR=16 -D INC=1 -D DATATYPE=F32 -o src/f32-gemminc/gen/f32-gemminc-8x16-minmax-avx512f-broadcast.c &
 
+################################ RISC-V Vector ################################
+tools/xngen src/f32-gemm/MrxLmulv-rvv.c.in -D MR=7 -D LMUL=4 -D ACTIVATION=LINEAR -D DATATYPE=F32 -o src/f32-gemm/gen/f32-gemm-7x4v-rvv.c &
+tools/xngen src/f32-gemm/MrxLmulv-rvv.c.in -D MR=7 -D LMUL=4 -D ACTIVATION=RELU   -D DATATYPE=F32 -o src/f32-gemm/gen/f32-gemm-7x4v-relu-rvv.c &
+tools/xngen src/f32-gemm/MrxLmulv-rvv.c.in -D MR=7 -D LMUL=4 -D ACTIVATION=MINMAX -D DATATYPE=F32 -o src/f32-gemm/gen/f32-gemm-7x4v-minmax-rvv.c &
```

Comment:
Add an MR=1 kernel.  When gemm-config is filled in, there is always a regular gemm, and a gemm1.
When implementing MR=1 you may want to do multiple accumulators (e.g. acc2).

Reply:
Agree.

Reply:
> When implementing MR=1 you may want to do multiple accumulators (e.g. acc2).

I think we might need to provide this using new `.in` file.